pkg/plugins/policies/core/matchers/testdata/egressmatchedpolicies/meshexternalservice/torules/01.golden.yaml

```diff
@@ -0,0 +1,27 @@
+ResourceRules:
+  meshexternalservice:mesh/mesh-1:name/mes:
```

Comment:
We're missing entry for `mesh:name/mesh-1`, you should add `Mesh` resource to `01.policies.yaml` to get the entry. See the example https://github.com/kumahq/kuma/blob/fca161a8bb09b00662a7b67a416060c019ac1975/pkg/plugins/policies/core/rules/testdata/resourcerules/mhr_with_ms_and_hostnames.input.yaml#L55-L54